app/helpers/content_block_helper.rb

```diff
@@ -0,0 +1,20 @@
+module ContentBlockHelper
+  include Sufia::ContentBlockHelperBehavior
+
+  def displayable_content_block(content_block, **options)
+    return if content_block.value.blank?
+    content_tag :div, content_block.value, options
+  end
+
+  def editable_content_block(content_block)
+    capture do
+      concat form_for([sufia, content_block]) { |f|
+        concat f.label :value, content_block.name.titleize
+        concat f.text_area :value, class: "tinymce", rows: 20, cols: 120
+        concat f.submit 'Save', class: "btn btn-primary"
+      }
+      concat tinymce_assets
```

Comment:
OK, my current impl does away with the duplication but does not explore playing nicer with the asset pipeline. Will push what I have soon.